assets/js/portalOnReady.js

```diff
@@ -183,6 +183,11 @@ $(document).ready(function () {
         }
     });
 
+    const announcement = document.getElementById("siteAnnouncement")
+const announcementcloseButton = document.getElementById("close-announcement")
+
+announcementcloseButton.onclick = function() { announcement.remove() };
+
```

Comment:
```suggestion
    const announcement = document.getElementById("siteAnnouncement")
    const announcementcloseButton = document.getElementById("close-announcement")

    announcementcloseButton.onclick = function() { announcement.remove() };

```

Sorry for being nitpicky, just want the indents to be correct/clean